camayoc/tests/qpc/api/v1/sources/test_sources_common.py

```diff
@@ -67,6 +69,60 @@ def test_create(shared_client, cleanup, scan_host, src_type):
     assert_matches_server(src)
 
 
+@pytest.mark.parametrize('scan_host', CREATE_DATA)
+def test_create_exclude_hosts(shared_client, cleanup, scan_host):
+    """Create a source with the excluded hosts option.
+
+    :id: 40365ec0-a8b5-4dc9-b7b4-dad826602d66
+    :description: Create a network source with some excluded host IPs.
+    :steps:
+        1) Create host credential
+        2) Send POST with data to create the source using the credential to
+           the source endpoint, with the exclude hosts option.
+    :expectedresults: A new source entry is created with the data.
+    """
+    source = gen_valid_source(
+            cleanup,
+            'network',
+            host='1.1.1.1',
+            exclude_host=scan_host,
+    )
+    print(source.list().json())
```

Comment:
I think it would be better to assert for the returned value instead of calling print. Or just call `.json` (without the print) to make sure an exception won't be raised if that is the intention.

Reply:
ah, I left this line as debug at the time, since gen_valid_source contains its own internal assertion.